bitnami/fluentd/templates/servicemonitor.yaml

```diff
@@ -26,8 +26,8 @@ spec:
       {{- include "common.tplvalues.render" (dict "value" .Values.metrics.serviceMonitor.selector "context" $) | nindent 6 }}
       {{- end }}
   endpoints:
-    - port: tcp-metrics
-      path: "/metrics"
+    - port: {{ default "tcp-metrics" (default (index .Values.metrics.service.annotations "prometheus.io/port") .Values.metrics.service.port) | quote}}
```

Comment:
is it necessary to override the tcp-metrics string by `metrics.service.port`? In principle this should be the same, right?

Reply:
Fixed . thanks for the suggestion